povm_toolbox/library/randomized_projective_measurements.py

```diff
@@ -170,7 +170,7 @@ def _build_qc(self) -> QuantumCircuit:
             qc.sx(qubit=i)
             qc.rz(np.pi - self._qc_theta[i], qubit=i)
             qc.sx(qubit=i)
-            qc.rz(3 * np.pi, qubit=i)
+            # qc.rz(3 * np.pi, qubit=i)
```

Comment:
Why is this necessary?

Reply:
The POVM effects (and hence the outcome probabilities) are actually the same if we have this final gate or not, so I thought we could just remove it

Reply:
But in that case, let's outright remove the line of code :+1: